improver/regrid/landsea.py

```diff
@@ -245,17 +257,16 @@ def __init__(self, extrapolation_mode="nanmask", vicinity_radius=25000.0):
         self.regridder = Nearest(extrapolation_mode=extrapolation_mode)
         self.vicinity = OccurrenceWithinVicinity(vicinity_radius)
 
-    def correct_where_input_true(self, selector_val):
+    def correct_where_input_true(self, selector_val: int):
```

Comment:
If a method or function returns `None` this can be annotated as `-> None`. I like to do this.

Reply:
Done/Thanks